src/PDE/CompFlow/DGCompFlow.hpp

```diff
@@ -820,13 +820,14 @@ class CompFlow {
                     tk::real, tk::real, tk::real, tk::real,
                     const std::array< tk::real, 3 >& )
     {
-      const auto p_farfield = g_inputdeck.get< tag::bc, tag::p_farfield >();
+      auto fp =
+        g_inputdeck.get< tag::param, eq, tag::farfield_pressure >()[ system ];
```

Comment:
And this how we access the value from the compflow parameter block. IT is now a vector so we also must index it by the `system` index, i.e., _"which compflow block's farfield pressure you want"?_